src/features/rewards/walletPanel/index.tsx

```diff
@@ -0,0 +1,207 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+// Helpers
+import { getLocale } from '../../../helpers'
+
+// Styled Components
+import {
+  StyledWrapper,
+  StyledAttentionScore,
+  StyledAttentionScoreTitle,
+  StyledContainer,
+  StyledScoreWrapper,
+  StyledControlsWrapper,
+  StyledDonateText,
+  StyledIcon,
+  StyledDonateWrapper,
+  StyledToggleWrapper,
+  StyledSelectWrapper
+} from './style'
+
+// Components
+import {
+  Grid,
+  Column,
+  Select,
+  Toggle,
+  ButtonSecondary
+} from '../../../components'
+import { Tokens } from '../'
+import Profile, { Provider } from '../profile/index'
+import ToggleTips from '../toggleTips/index'
+
+// Assets
+const alertIcon = require('./assets/alert')
+const monthlyIcon = require('./assets/monthly')
+
+export interface Props {
+  id?: string
+  platform?: Provider
+  publisherImg?: string
+  publisherName?: string
+  isVerified?: boolean
+  attentionScore?: string | null
+  donationAction: () => void
+  onToggleTips: () => void
+}
+
+interface State {
+  tipsEnabled: boolean
+  includeInAuto: boolean
+  monthlyAmount: number | null
+}
+
+export default class WalletPanel extends React.PureComponent<Props, State> {
+  constructor (props: Props) {
+    super(props)
+    this.state = {
+      tipsEnabled: true,
+      includeInAuto: true,
+      monthlyAmount: null
+    }
+  }
+
+  onIncludeInAuto = () => {
+    this.setState({
+      includeInAuto: !this.state.includeInAuto
+    })
+  }
+
+  publisherInfo () {
+    return (
+      <StyledWrapper>
+        <Profile
+          type={'big'}
+          title={'Bart Baker'}
+          provider={this.props.platform}
+          src={this.props.publisherImg}
+          verified={this.props.isVerified}
+        />
+      </StyledWrapper>
+    )
+  }
+
+  getSelectTheme () {
+    return {
+      border: 'none',
+      padding: '0px',
+      background: 'inherit'
+    }
+  }
+
+  donationDropDown () {
+    return (
+      <StyledSelectWrapper>
+        <Select theme={this.getSelectTheme()}>
+          <div data-value='5'>
```

Comment:
This should be defined in the props, so that we can dynamically change this select.